### ISSUE TITLE
Image coordinate system

### ISSUE THREAD

**Issue by SubChange** [cdcseacave/openMVS#1070]
In colmap Docs, It says that:

> Note that by convention the upper left corner of an image has coordinate (0, 0) and the center of the upper left most pixel has coordinate (0.5, 0.5). If you must import features for large image collections, it is much more efficient to directly access the database with your favorite scripting language (see [Database Format](https://colmap.github.io/database.html#database-format)).

However, In OpenMVS, it says:

> The projection in image coordinates uses the convention that the center of a pixel is defined at integer coordinates, i.e. the center is at (0, 0) and the top left corner is at (-0.5, -0.5). All matrices are expected in row-major format.

In apps/InterfaceCOLMAP, I don't see the conversion about the 0.5 pixel shift, Is this cause any errors?

**Comment by cdcseacave** [comment]
good point, I did not notice that the pixel center convention is different, thank you
feel free to add a fix if you can, if not I'll try to find some time next weekend

**Comment by SubChange** [comment]
> good point, I did not notice that the pixel center convention is different, thank you feel free to add a fix if you can, if not I'll try to find some time next weekend

Thank you very much, I ‘d like to try but I am not good at c++. So, I'd greatly appriciate it if you can correct that.

**Comment by cdcseacave** [comment]
should be solved here, pls give it a try

**Comment by cdcseacave** [comment]
Fixed in commit f711e4ab906818b3b7e2d50f0d1fd56e5d200602.
